.travis.yml

```diff
@@ -12,6 +12,9 @@ python:
 service:
 - docker
 install:
+- python -m pip install --upgrade pip
+- python -m pip install --upgrade setuptools>=41.0.0
+- python -m pip install --upgrade tensorflow==2.0.0
```

Comment:
Following up the discussion at https://github.com/sql-machine-learning/models/pull/46#discussion_r364054604.

Hi @shendiaomo, Please allow me to explain.

I understand that line 15 and line 16 might be necessary due to pip packages in Travis CI is outdated. However, I am questioning the nececity of line 17.